### DISCUSSION TITLE
Was the identification of "front-end JavaScript libraries with known security vulnerabilities" removed from LightHouse?

### DISCUSSION THREAD

**Discussion by cqueern** [GoogleChrome/lighthouse#14961] [Q&A] [answered]
There used to be an output in the Best Practices section of a Lighthouse report that detailed whether the page included vulnerable 3rd party dependencies using a very nice integration with Snyk called "Includes front-end JavaScript libraries with known security vulnerabilities". I don't see it in lighthouse 10.0.1. 

This page https://developer.chrome.com/docs/lighthouse/best-practices/no-vulnerable-libraries/ suggests the feature still should be present, and I've tried multiple sites in Chrome and Edge but still can't find that output.

Is this expected behavior?

**Comment by adamraine** [comment, marked as answer]
It was removed in https://github.com/GoogleChrome/lighthouse/releases/tag/v10.0.0

**Reply by cqueern** [reply]
Thanks Adam. I hadn't seen that.

Sad to see it go but the [logic for the decision](https://github.com/GoogleChrome/lighthouse/pull/14194#issue-1296278432) seems sound.
